content/product-engineering/direction.md

```diff
@@ -0,0 +1,17 @@
+# Product and engineering direction
```

Comment:
Maybe rename this file to strategy.md? Not super important but it's more consistent.